alvr/client_openxr/src/extra_extensions/mod.rs

```diff
@@ -42,7 +42,7 @@ fn get_props<G, T>(
     let instance = session.instance();
 
     let mut props = default_struct;
-    let mut system_properties = sys::SystemProperties::out((&mut props as *mut T).cast());
+    let mut system_properties = sys::SystemProperties::out((&raw mut props).cast());
```

Comment:
Good use of the new `raw` keyword. I was going to go through myself adding it when necessary at some point